frontend/src/components/DefenceBox/DefenceMechanism.css

```diff
@@ -1,106 +1,164 @@
-.defence-mechanism {
-	cursor: default;
+.defence-mechanism-fieldset {
+	min-height: fit-content;
+	margin: 0.1em;
+	border: 0.25rem solid var(--control-config-border);
+	border-radius: 0.5rem;
+}
+
+/* the name of the fieldset */
+.defence-mechanism-legend {
+	padding: 0.3em;
 }
 
-.defence-mechanism > * {
-	padding: 0.75rem;
+/* the more button to expand the details */
+.defence-mechanism-summary {
+	width: fit-content;
+	padding: 0.5rem;
+	border: 0.125rem outset var(--control-config-border);
+	border-radius: 0.25rem;
 }
 
 .defence-mechanism > summary {
-	position: relative;
 	font-size: 1rem;
 }
 
-/*
- * Modified from https://www.w3schools.com/howto/howto_css_switch.asp
-*/
+/* remove the default triangle on the summary element */
+.defence-mechanism-fieldset details > summary {
+	list-style: none;
+}
 
-/* The switch - the box around the slider */
-.defence-mechanism label.switch {
+.defence-mechanism-fieldset details > summary::-webkit-details-marker {
+	display: none;
+}
+
+/* style for when details element is expanded */
+.defence-mechanism-fieldset details[open] > summary {
+	border-style: inset;
+}
+
+/* the toggle and label combined */
+.defence-mechanism-form {
 	position: absolute;
-	top: 0.75rem;
-	right: 0.75rem;
-	width: 2rem;
-	height: 1rem;
+	right: 1em;
 }
 
-.defence-mechanism label.switch:focus-within {
-	border-radius: 0.5rem;
-	outline-style: auto;
+.defence-mechanism {
+	cursor: default;
 }
 
-/* Hide default HTML checkbox */
-.defence-mechanism label.switch > input {
-	width: 0;
-	height: 0;
-	opacity: 0;
+.defence-mechanism .info-box :first-child {
+	margin-top: 0;
 }
 
-/* The slider */
-.defence-mechanism label.switch > .slider {
-	--slider-border-width: 0.125rem;
+/* the expanded description and further controls */
+.defence-mechanism .info-box {
```

Comment:
another case of removing a comment and improving the classname. Maybe `expanded-box` or `expanded-description-and-config-value`. I'd have to give it some more thought.

Reply:
it's the same className that it currently is on dev, so didn't want to change that around unnecessarily, especially since we don't really have consistent naming convention throughout

Reply:
very fair, not strictly relevant to the PR. Happy to leave as is, but would prefer no comment.